python/cudf/cudf/tests/test_csv.py

```diff
@@ -150,8 +150,8 @@ def make_all_numeric_extremes_dataframe():
         np_type = pdf_dtypes[gdf_dtype]
         if np.issubdtype(np_type, np.integer):
             itype = np.iinfo(np_type)
-            extremes = [0, +1, -1, itype.min, itype.max]
-            df[gdf_dtype] = np.array(extremes * 4, dtype=np_type)[:20]
+            extremes = [itype.min, itype.max]
```

Comment:
Could you elaborate? Is this a task you want to accomplish in this PR?

Reply:
`np.uint8(-1)` in particular raises a deprecation warning. But the reason I gave up on this test was because I couldn't figure out why this was happening:

```python
In [2]: np.array([-1]).astype("uint64")
Out[2]: array([18446744073709551615], dtype=uint64)

In [3]: np.array([18446744073709551615]).astype("uint64")
Out[3]: array([18446744073709551615], dtype=uint64)

In [4]: np.array([-1, 18446744073709551615]).astype("uint64")
<ipython-input-4-03014ed268fc>:1: RuntimeWarning: invalid value encountered in cast
  np.array([-1, 18446744073709551615]).astype("uint64")
Out[4]: array([18446744073709551615,                    0], dtype=uint64)
```

I've gone ahead and filtered out that warning from this test.